src/site/content/en/blog/preload-responsive-images/index.md

```diff
@@ -187,6 +192,12 @@ Because responsive preload has no notion of "order" or "first match", the breakp
 <link rel="preload" href="large_cat.jpg" as="image" media="(min-width: 800.1px)">
 ```
 
+### Preload and `type`
+
+The `<picture>` element also supports matching on the first `type`, to allow different image formats to be provided and the browser to pick the first image format it supports. This use case is not currently supported with preload.
+
+For sites using this, avoiding preload is the best option, and instead having the [preload scanner](/preload-scanner/) pick these up from the `<picture>` and `<source>` elements instead. This is a best practice anyway, particular with the support of [Priority Hints](./priority-hints/) which allows the appropriate image to be prioritised in a better manner than preload alone.
```

Comment:
Should there be a dot in that URL? All the URLs for blog posts are flattened AFAIK, so it should just be `/priority-hints/`.

Reply:
```suggestion
For sites using this, avoiding preload is the best option, and instead having the [preload scanner](/preload-scanner/) pick these up from the `<picture>` and `<source>` elements instead. This is a best practice anyway, particular with the support of [Priority Hints](/priority-hints/) which allows the appropriate image to be prioritised in a better manner than preload alone.
```